hknweb/exams/templates/exams/index.html

```diff
@@ -21,12 +34,21 @@
 
 {% block content %}
 <div id="actual-contents">
+    <center>
+        <select id="selectSemestercourses" style="width: 300px">
+            <option selected disabled="true">--- Search for Course ---</option>
+            {% for coursesemester in searchCourses %}
+            <option value="/exams/course/{{ coursesemester.course.department.abbreviated_name }}/{{ coursesemester.course.number }}/?term={{ coursesemester.semester.semester }}">{{coursesemester.semester.semester}} {{coursesemester.course.department.abbreviated_name}} {{coursesemester.course.number}}</option>
+            {% endfor %}
+        </select>
+    </center>
+    <br>
     Browse by department:
 
     <h1>Computer Science</h1>
     <ul>
         {% for course in courses %}
-            {% if course.department.name == 'CS' %}
+            {% if course.department.abbreviated_name == 'CS' %}
```

Comment:
This is hardcoded in; let's refactor this so it's more robust?

Reply:
Done